src/github.com/getlantern/chained/dialer.go

```diff
@@ -88,7 +89,7 @@ func checkCONNECTResponse(r *bufio.Reader, req *http.Request) error {
 	if err != nil {
 		return fmt.Errorf("Error reading CONNECT response: %s", err)
 	}
-	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+	if resp.StatusCode < http.StatusOK || resp.StatusCode > 299 {
```

Comment:
I actually think that would be confusing too, since reading the `StatusOK` constant plus a number looks like trying to add a number to something that means a status response, and we're comparing the ordering of status codes with integers, which is already awkward. 

Either way I don't have a strong feeling on changing `200` by `http.StatusOK` in this particular case, since the `<` comparison is already a trick based on the actual integer value of `http.StatusOK`. Maybe these kind of constants are better suited for (in)equalities. So, if you feel this is more awkward than it was we may as well use the previous comparison.